docs/v4-migration-doc.md

```diff
@@ -136,6 +116,16 @@ kube-prometheus-stack:
             sourceLabels: [__name__]
 ```
 
+#### Apply missing Custom Resource Definition for OpenTelemetry Operator
+
+**When??** If you already uses OpenTelemetry Operator, and it is in version before `0.42.0`
+
+Please apply the following command on your cluster:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/open-telemetry/opentelemetry-helm-charts/opentelemetry-operator-0.44.0/charts/opentelemetry-operator/crds/crd-opentelemetry.io_opampbridges.yaml
+```
+
```

Comment:
If you want to add something here, it should instead go to https://github.com/SumoLogic/sumologic-documentation/blob/main/docs/send-data/kubernetes/v4/how-to-upgrade.md.

Reply:
We should then link from this document to the Sumo Logic. Now we have duplication which is easy to keep unsynced

Reply:
Yes, now done with https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3477 merged.

Reply:
https://github.com/SumoLogic/sumologic-documentation/pull/3489/files

Reply:
```suggestion
```